ar2_arm:
  hardware_interface:
    loop_hz: 20 # hz
    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_6
    hardware: 'rosserial'
    rosserial:
      topic: '/arm_hardware_interface'
    serial:
      port: '/dev/ttyACM0'
      baudrate: 57600